android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -193,6 +193,7 @@ public void joinMeeting(
       if(paramMap.hasKey("noShare")) opts.no_share = paramMap.getBoolean("noShare");
       if(paramMap.hasKey("noTitlebar")) opts.no_titlebar = paramMap.getBoolean("noTitlebar");
       if(paramMap.hasKey("customMeetingId")) opts.custom_meeting_id = paramMap.getString("customMeetingId");
+      if(paramMap.hasKey("webinarToken")) opts.webinar_token = paramMap.getString("webinarToken");
```

Comment:
there should be only two changes in this pr:
1. this one-line is one change
2. the second change should be in `index.ts` in `RNZoomUsJoinMeetingParams`

Search for `noTitlebar` in the codebase for an example.

Reply:
actually `webinarToken` is already in `index.ts`. you can check in line 85 which it's notified that webinarToken is only for iOS.

Reply:
Got it, it is then simply moving this line up so it is clear that the support is for both ios and android